pages/_app.tsx

```diff
@@ -81,4 +58,4 @@ export default function App({ Component, pageProps }) {
       </div>
     </>
   );
-}
\ No newline at end of file
+}
```

Comment:
Do you feel strongly about removing the SocialShare by default and forcing the pages always to include it? 
If yes, it's fine.

If not, I would consider providing this functionality by default and make it overridable on demand.

Reply:
I will ponder about this later. It's a minor concern and non-blocker in any way.